stdlib/3/typing.pyi

```diff
@@ -663,3 +663,15 @@ if sys.version_info >= (3, 7):
         def __eq__(self, other: Any) -> bool: ...
         def __hash__(self) -> int: ...
         def __repr__(self) -> str: ...
+
+class _NoneType:
```

Comment:
The class could be final, since it's not subclassable at runtime.

Reply:
The final decorator is available only for 3.8 and newer, but `_NoneType` needs to be available regardless of version.

Reply:
But it's always in `typing_extensions`.

Reply:
Yes, but I don't think we want to import `typing_extensions` from `typing`. Is the consensus that I should move this out of `typing.pyi` and into `_typeshed.pyi`? That sounds inconsistent to me given that other types like this (e.g. `_TypedDict`) are defined in `typing.pyi`, but if that's what you'd prefer, I can move it. And then I could add both the `final` decorator and change `__bool__` to return `Literal[False]`.